### PR TITLE
updating image, small grammar changes to /kommander/1.2/licensing/ind…

### PR DESCRIPTION
**Pull request by msdolbey**
…ex.md

## Jira Ticket
Related to: https://jira.d2iq.com/browse/D2IQ-71353

## Description of changes being made
Updated the error state image for the license page and just did some small grammar changes.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.2/licensing/index.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ menuWeight: 6
...
 ## Enter a Valid License Key
 
 After you have downloaded the license, an administrator must add it to Kommander.
 
 In the Kommander UI, do the following:
 
-1. Select **Global** in the header drop-down.
-2. Select **Administration** > **Licensing**.
+1. Select **Global** in the workspace header drop-down.
+2. Along the left rail, select **Administration** > **Licensing**.
```
Comment (natmegs) [inline pages/ksphere/kommander/1.2/licensing/index.md:23]: I think we usually refer to this as side menu but this works too :)

**Review by natmegs — APPROVED** [review]

File: pages/ksphere/kommander/1.2/licensing/index.md (hunk 1 above)
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/licensing/index.md:23]:
```suggestion
2. In the side menu, select **Administration** > **Licensing**.
```

Comment (msdolbey) [reply pages/ksphere/kommander/1.2/licensing/index.md]: Thanks @natmegs

File: pages/ksphere/kommander/1.2/licensing/index.md [hunk 2]
```diff
@@ -9,20 +9,20 @@ menuWeight: 6
...
 3. Select **+ Add License**.
-4. Paste your license content in and select Add.
+4. Paste your license content in the textbox and select Add.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/licensing/index.md:25]:
```suggestion
4. Paste your license content in the textbox and select **Add**.
```

**Review by russurquhart1 — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request merged**